app/views/devise/passwords/new.html.haml

```diff
@@ -1,9 +1,23 @@
-%article.content-listing
-  %h2 Forgot your password?
-  = simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
-    = f.error_notification
-    .form-inputs
-      = f.input :email, required: true, autofocus: true
-    .form-actions
-      = f.button :submit, "Send me reset password instructions"
-  = render "devise/shared/links"
+%article.sign-in
+  %h1 Forgot your password?
+  %section.sign-in-form
+    = simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
+      = devise_error_messages!
+
+
+      %p
+        = f.input :email, required: true, autofocus: true
+      .form-footer-actions
+        %p
+          = f.button :submit, "Send me reset password instructions", role: 'button'
+
+    .sign-in-footer-links
+      - if devise_mapping.confirmable? && controller_name != 'confirmations'
+        = link_to "Didn't receive confirmation instructions?", new_confirmation_path(resource_name)
```

Comment:
@ltankey I had a look at how that may work. The issue is that the type of links changes depending on which page you are on. EG there 'Remember password' is unique to the Sign-in page, whereas the Sign-in link is redundant on the Sign-in page, but should appear on password rest and Sign-up page etc. 

TLDR: I believe the simplest way is to not use a `shared/links` call for now